pkg/getter/getter.go

```diff
@@ -38,6 +39,11 @@ type options struct {
 	userAgent             string
 }
 
+type ChartResponse struct {
+	ChartContent *bytes.Buffer
```

Comment:
can we rename this to just "Content" since the type has chart in the name already?